ssl/record/rec_layer_s3.c

```diff
@@ -533,7 +533,7 @@ int ssl_release_record(SSL_CONNECTION *s, TLS_RECORD *rr, size_t length)
  * Return up to 'len' payload bytes received in 'type' records.
  * 'type' is one of the following:
  *
- *   -  SSL3_RT_HANDSHAKE (when ssl3_get_message calls us)
+ *   -  SSL3_RT_HANDSHAKE (when tls_get_message_header and tls_get_message_body calls us)
```

Comment:
Adding a second function makes the subject of the verb "call" plural, so this should be "call us" not "calls us"

Reply:
Okay.